src/openrct2/park/ParkFile.h

```diff
@@ -8,10 +8,10 @@ struct ObjectRepositoryItem;
 namespace OpenRCT2
 {
     // Current version that is saved.
-    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 0xA;
+    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 0xB;
```

Comment:
Why did you increase the version number?

Reply:
Because saves made with this fix can't be opened in versions without the fix. Saves made without the fix can be opened with the fix though of course.

Reply:
Well, that’s odd, because I was under the impression that it saved 8-bit and 16-bit integers as 32-bit or 64-bit ones. Perhaps @IntelOrca can shine a bit of light on this?

Reply:
Yes they are saved as 32bit integers but it will still cause a panic if you try load the new save in the old version as it will say the size is too large for the type. This up rev is correct.